cpu/cc2538/include/cpu_conf.h

```diff
@@ -42,6 +42,15 @@ extern "C" {
 #define CPU_FLASH_BASE                  FLASH_BASE
 /** @} */
 
+/**
+￼ * @brief   Bit-Band configuration
+￼ * @{
+￼ */
+#ifdef SRAM_BB_BASE
+#define CPU_HAS_BITBAND 1
+#endif
```

Comment:
Here `SRRAM_BB_BASE` is not coming from the vendor files. I suggest to just manually define it to `1`, so that it is obvious we don't rely on vendor info here.
```suggestion
#define CPU_HAS_BITBAND 1
```

Reply:
Ok, moved it to the common CPU config block above